3.Tests/UI/Hooks/Hooks.cs

```diff
@@ -61,6 +58,23 @@ public void CreateProject()
         _scenarioContext[ConfigModel.CurrentProjectPayload] = projectModel;
     }
 
+    [AfterScenario]
+    public void CaptureScreenshot()
+    {
+        if (_scenarioContext.TestError != null)
+        {
+            Screenshot image = ((ITakesScreenshot)GenericWebDriver.Instance).GetScreenshot();
+            string fileName = $"{_scenarioContext.ScenarioInfo.Title}_{DateTime.Now}";
+            fileName = string.Join(" ", fileName.Split().Select(word => char.ToUpper(word[0]) + word.Substring(1)));
+            fileName = fileName.Replace(" ", "");
+            fileName = fileName.Replace("/", "");
+            fileName = fileName.Replace(":", "");
+            image.SaveAsFile($"../../../Assets/{fileName}.png", ScreenshotImageFormat.Png);
```

Comment:
I think it is better to create a new folder for each test execution, and there you can store the screenshots for the whole test run. You are also missing to attach the screenshot to the LivingDoc report, is it going to be done on another PR?

Reply:
hi Sebastian, yeah in order to touch just the yaml file for the reports it´s going to be in the valeria/reports branch, and for the screenshots just this branch